packages/eslint-plugin/src/rules/restrict-template-expressions.ts

```diff
@@ -62,7 +62,19 @@ export default createRule<Options, MessageId>({
     docs: {
       description:
         'Enforce template literal expressions to be of `string` type',
-      recommended: 'recommended',
+      recommended: {
+        recommended: true,
+        strict: [
+          {
+            allowAny: false,
+            allowBoolean: false,
+            allowNullish: false,
+            allowNumber: false,
```

Comment:
Is this intended? If so, should this doc be updated?

> This rule reports on values used in a template literal string that aren't strings, numbers, or BigInts, optionally allowing other data types that provide useful stringification results.

Reply:
If you think there's a small typo in the docs like that, we'd welcome a PR :) thanks